sdk/core/azure-core-amqp/test/ut/session_tests.cpp

```diff
@@ -30,33 +30,34 @@
 #endif
 #endif // AZ_PLATFORM_POSIX/AZ_PLATFORM_WINDOWS
 
-class TestSessions : public testing::Test {
-protected:
-  void SetUp() override {}
-  void TearDown() override {}
-};
+namespace Azure { namespace Core { namespace Amqp { namespace Tests {
+  class TestSessions : public testing::Test {
+  protected:
+    void SetUp() override {}
+    void TearDown() override {}
+  };
 
-using namespace Azure::Core::Amqp::_internal;
-using namespace Azure::Core::Amqp;
+  using namespace Azure::Core::Amqp::_internal;
+  using namespace Azure::Core::Amqp;
 
 #if !defined(AZ_PLATFORM_MAC)
-TEST_F(TestSessions, SimpleSession)
-{
-
-  // Create a connection
-  Azure::Core::Amqp::_internal::Connection connection("localhost", {});
+  TEST_F(TestSessions, SimpleSession)
   {
-    // Create a session
-    Session session(connection, nullptr);
-  }
 
-  {
-    // Create two sessions
-    Session session1(connection, nullptr);
-    Session session2(connection, nullptr);
+    // Create a connection
+    Azure::Core::Amqp::_internal::Connection connection("localhost", nullptr, {});
+    {
+      // Create a session
+      Session session{connection.CreateSession()};
+    }
 
-    session1.End("", "");
-  }
+    {
+      // Create two sessions
+      Session session1{connection.CreateSession({})};
+      Session session2{connection.CreateSession({})};
+
+      session1.End("", "");
+    }
 
 #if 0
```

Comment:
Why keep this code?  Is it coming back?

Reply:
No reason. It was test collateral that GitHub CoPilot hallucinated and I thought it might be helpful at some point.